components/places/sql/create_shared_triggers.sql

```diff
@@ -61,12 +61,15 @@ BEGIN
     WHERE NEW.guid = 'root________'
       AND NEW.parent IS NOT NULL;
 
-    SELECT throw('update: item without parent')
+    SELECT throw(format(
+        'update: nonexistent parent: old_parent=%d, new_parent=%d, operation=%q',
+        OLD.parent,
+        NEW.parent,
+        CASE WHEN NEW.syncChangeCounter > 0 THEN 'sync' ELSE 'user' END
+    ))
     WHERE NEW.guid <> 'root________'
-      AND NEW.parent IS NULL;
```

Comment:
This might be worth spelling out in the PR description, but why are we rolling back this change? I assumed we were keeping it and continuing to research why it's problematic. I'm not necessarily opposed to making this change, I just think an explanation should be provided.

Reply:
ah yeah I was going to detail a little of why would should add back in and added the relevant comment in the `ON CONFLICT` section but i'll also detail it in the description too!